doc/lsst.meas.algorithms/creating-a-reference-catalog.rst

```diff
@@ -128,14 +128,54 @@ Monitor the log file in a new terminal with:
 
     tail -f convert-gaia.log
 
-Check the log ouput after several hours: ``ConvertReferenceCatalogTask`` reports progress in 1% intervals.
+Check the log output after several hours: ``ConvertReferenceCatalogTask`` reports progress in 1% intervals.
 
 .. _lsst.meas.algorithms-refcat-ingest:
 
 4. Ingest the files into the butler
 ===================================
 
-When ``convertReferenceCatalog`` has finished, it will print the two commands you need to run to register the new refcat dataset type and ingest your converted output into it.
+When ``convertReferenceCatalog`` has finished, a new directory (named ``gaia-refcat/`` in the example above) will now exist containing all reference catalogs in the LSST format.
+
+.. note::
+
+    If you have an already existing converted reference catalog on disk (for example, the PS1 or Gaia DR2 catalogs that were used in gen2 butlers), you will first need to create a suitable filename to htm7-index astropy lookup table before running the commands in the remainder of this section.
+
+    This is an example script that constructs conversion lookup tables for all converted FITS files in the `gaiadr2` and `panstarrsps1` directories, outputting to the `outdir` directory:
+
+    .. code-block:: python
+
+        import os
+        import glob
+        import astropy.table
+
+        refcat_dirs = [gaiadr2, panstarrsps1]
```

Comment:
These need to be strings.

Reply:
They're meant to be objects, that the user has already assigned elsewhere. I hoped the intro text to this code block explained that. As this code is intended to be used on other machines, where the refcats may be saved in a non-standard location, I thought this was the safest bet.

If it would help, I could add some dummy lines above in the script along the lines of:

```
gaiadr2 = "/path/to/my/gaia/dir"
panstarrsps2 = "/path/to/my/panstarrs/dir"
```